app/pages/dashboard/contextCreator.js

```diff
@@ -0,0 +1,10 @@
+import manifest from './manifest.json';
+
+export const getOrgDashboardContext = data => ({
+  ...manifest,
+  columnInfo: manifest.columnInfo ? manifest.columnInfo.map(heading => ({
+    data: heading.data || '',
+    link: heading.link || false,
+  })) : [],
+  data: data ? data.map(row => [row.name || '', row.odsCode || '']) : [],
```

Comment:
Is this what Object.enteries does for us? Converts an object to an array or arrays. Or do we need this to replace with empty strings if it doesn't exist

Reply:
we need empty strings incase there is missing data - we need to preserve the space